From 49cd00921746c634a03e141287cd507f7309dc81 Mon Sep 17 00:00:00 2001
From: Antonio Ospite <ao2@ao2.it>
Date: Tue, 16 Oct 2018 18:42:16 +0200
Subject: [PATCH 34/53] shell: Update configure.ac with suggestions from
 autoupdate

Apply the changes suggested by running autoupdate on the source
repository:

1. Properly quote AC_INIT arguments.

2. Use AC_USE_SYSTEM_EXTENSIONS instead of AC_GNU_SOURCE. The former is
   a superset of the latter, and enables more options, see
   https://www.gnu.org/software/autoconf/manual/autoconf-2.60/html_node/Posix-Variants.html

Signed-off-by: Antonio Ospite <ao2@ao2.it>
Signed-off-by: Herbert Xu <herbert@gondor.apana.org.au>
---
 configure.ac | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index 4829288..594fb42 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1,4 +1,4 @@
-AC_INIT(dash, 0.5.10.2)
+AC_INIT([dash],[0.5.10.2])
 AM_INIT_AUTOMAKE([foreign subdir-objects])
 AC_CONFIG_SRCDIR([src/main.c])
 
@@ -6,7 +6,7 @@ AC_CONFIG_HEADERS(config.h)
 
 dnl Checks for programs.
 AC_PROG_CC
-AC_GNU_SOURCE
+AC_USE_SYSTEM_EXTENSIONS
 AC_PROG_YACC
 
 AC_MSG_CHECKING([for build system compiler])
-- 
2.25.0

